code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -223,49 +223,51 @@
+			It is sometimes claimed that particularly desperate Eridanian dregs came up with the current recipe for this drink."
 	desc_fluff = DRINK_FLUFF_GETMORE
 	icon_state = "whiskeybottle"
 	center_of_mass = list("x"=16, "y"=4)
 	reagents_to_add = list(/decl/reagent/alcohol/fireball = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/vodka
 	name = "Martian 50% Premium"
-	desc = "Only potatoes grown in real imported Martian soil may be used for this premium vodka. Made by Silverport, drunk by Zavodskoi."
+	desc = "Only potatoes grown in real imported Martian soil may be used for this premium vodka (imports of Martian soil may have stopped). Made by Silverport, drunk by Zavodskoi."
 	desc_fluff = DRINK_FLUFF_SILVERPORT
 	icon_state = "vodkabottle"
 	center_of_mass = list("x"=17, "y"=4)
 	reagents_to_add = list(/decl/reagent/alcohol/vodka = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/vodka/mushroom
-	name = "Magnitogorsk Import mushroom vodka"
-	desc = "A mushroom-based vodka imported from the breweries of Himeo. Drinking too much of this will result in a personal permanent revolution."
+	name = "Inverkeithing Import mushroom vodka"
+	desc = "A mushroom-based vodka imported from the breweries of Inverkeithing on Himeo. Drinking too much of this will result in a personal permanent revolution."
 	desc_fluff = "Vodka made from mushrooms is a local favourite on Himeo, due to the ease with which mushrooms can be grown under the planet's surface. This bottle is \
-	from the Vodnik Breweries Syndicate in Magnitogorsk, arguably the most famous brewery on Himeo (though fans of the Coal Canary Syndicated Brewery in Highland City \
-	would disagree). Drinkers of the world, unite!"
+	from the Schwarzer Drache Breweries Syndicate in Inverkeithing, arguably the most famous brewery on Himeo due to its historical reputation. It is also the most famous brand \
+	of mushroom vodka among non-Himeans because of Inverkeithing's developing tourism industry. Drinkers of the world (and beyond), unite!"
 	icon_state = "mushroomvodkabottle"
 	reagents_to_add = list(/decl/reagent/alcohol/vodka/mushroom = 100)
 
 /obj/item/reagent_containers/food/drinks/bottle/tequila
-	name = "Chip's Guaranteed Quality tequila"
-	desc = "Made from premium petroleum distillates, pure thalidomide and other fine quality ingredients!"
+	name = "Nathan's Guaranteed Quality tequila"
+	desc = "Made from premium petroleum distillates, pure thalidomide and other fine quality ingredients! This particular line of tequila has Nathan Trasen's signature on the label and his approval, \
+	as can be commonly seen and heard in Getmore's many TV advertisements for this line. Astute observers may however note the absolute lack of emotion on Trasen's face while reciting his love for Getmore tequila on television."
```

Comment:
```suggestion
	as can be commonly seen and heard in Getmore's many advertisements for this line. Astute observers may however note the absolute lack of emotion on Trasen's face while reciting his love for the Getmore tequila."
```

Reply:
Not adding "the": you don't say "your love for the Coca Cola", you say "your love for Coca Cola".

Reply:
"Getmore tequila" isn't a brand name, though, "Coca Cola" is.